client/client_test.go

```diff
@@ -194,7 +193,7 @@ func (m mockAssumeRole) AssumeRole(ctx context.Context, params *sts.AssumeRoleIn
 func Test_Configure(t *testing.T) {
 	ctx := context.Background()
 	logger := hclog.New(&hclog.LoggerOptions{})
-	f, err := ioutil.TempFile("", "")
+	f, err := os.CreateTemp("", "")
```

Comment:
Is this a fix for something or making it more idiomatic?

Reply:
from go1.16 `io/ioutil` is deprecated, all these functions are in `os` package now